drizzlepac/haputils/hapcut_utils.py

```diff
@@ -0,0 +1,383 @@
+                                                  dataproduct_type="IMAGE",
+                                                  instrument_name="ACS*",
+                                                  project="HAP", 
+                                                  calib_level=3,
+                                                  obs_collection="HST") 
+
+    if verbose:
+        print("Performing query for WFC3 images.")
+    wfc3_query_table = Observations.query_criteria(coordinates=sky_coord,
+                                                   radius=radius,
+                                                   dataproduct_type="IMAGE",
+                                                   instrument_name="WFC3*",
+                                                   project="HAP", 
+                                                   calib_level=3,
+                                                   obs_collection="HST") 
+
+    query_table = vstack([acs_query_table, wfc3_query_table])
+
+    # Compute the limits of the cutout region
+    deg_cutout_size = cutout_size.to(u.deg)
+    ra_min = sky_coord.ra.value - deg_cutout_size.value[0]
+    ra_max = sky_coord.ra.value + deg_cutout_size.value[0]
+    dec_min = sky_coord.dec.value - deg_cutout_size.value[1]
+    dec_max = sky_coord.dec.value + deg_cutout_size.value[1]
+
+    # Filter the output as necessary to include only MVM filenames (MVM prefix: hst_skycell).
+    # Also, filter out images which are not actually in the requested cutout region as the
+    # archive search had to be done using a radius.
+    good_rows = []
+    updated_table = None
+    for i, old_row in enumerate(query_table):
+        if old_row["obs_id"].startswith("hst_skycell"):
+            if old_row["s_ra"] >= ra_min and old_row["s_ra"] <= ra_max and \
+               old_row["s_dec"] >= dec_min and old_row["s_dec"] <= dec_max:
+                good_rows.append(old_row)
+    updated_query_table = Table(rows=good_rows, names=query_table.colnames)
+
+    # Catch the case where no files are found which satisfy the criteria
+    if not updated_query_table:
+        print("WARNING: Query for objects within {} of {} returned NO RESULTS!".format(radius, sky_coord))
+        return updated_query_table
+
+    # Get the data product list associated with the elements of the table
+    if verbose:
+        print("Get the product list for all entries in the query table.")
+    dp_table = Observations.get_product_list(updated_query_table)
+
+    # Filter on MVM drizzled product
+    suffix = ["DRZ", "DRC"]
+    if verbose:
+        print("Filter the product list table for only {} filenames.".format(suffix))
+    filtered_dp_table = Observations.filter_products(dp_table,
+                                                     productSubGroupDescription=suffix,
+                                                     extension="fits")
+
+    if not filtered_dp_table:
+        print("WARNING: No drizzle product files (DRZ/DRC) found.")
```

Comment:
Another place where 'log.warning' would be appropriate.

Reply:
Addressed.